README.md

```diff
@@ -35,21 +35,174 @@ The package features a Pythonic high-performance implementation of the
   Super-Droplet Method (SDM) Monte-Carlo algorithm for representing collisional growth 
   ([Shima et al. 2009](https://rmets.onlinelibrary.wiley.com/doi/abs/10.1002/qj.441)), hence the name. 
 
+There is a growing set of example Jupyter notebooks exemplifying how to perform 
+  various types of calculations and simulations using PySDM.
+Most of the example notebooks reproduce resutls and plot from literature, see below for 
+  a list of examples and links to the notebooks (which can be either executed or viewed 
+  "in the cloud").
+
 PySDM has two alternative parallel number-crunching backends 
   available: multi-threaded CPU backend based on [Numba](http://numba.pydata.org/) 
   and GPU-resident backend built on top of [ThrustRTC](https://pypi.org/project/ThrustRTC/).
-The [`Numba`](https://open-atmos.github.io/PySDM/backends/numba/numba.html) backend (aliased ``CPU``) features multi-threaded parallelism for 
+The [`Numba`](https://open-atmos.github.io/PySDM/PySDM/backends/numba/numba.html) backend (aliased ``CPU``) features multi-threaded parallelism for 
   multi-core CPUs, it uses the just-in-time compilation technique based on the LLVM infrastructure.
-The [`ThrustRTC`](https://open-atmos.github.io/PySDM/backends/thrustRTC/thrustRTC.html) backend (aliased ``GPU``) offers GPU-resident operation of PySDM
+The [`ThrustRTC`](https://open-atmos.github.io/PySDM/PySDM/backends/thrustRTC/thrustRTC.html) backend (aliased ``GPU``) offers GPU-resident operation of PySDM
   leveraging the [SIMT](https://en.wikipedia.org/wiki/Single_instruction,_multiple_threads) 
   parallelisation model. 
 Using the ``GPU`` backend requires nVidia hardware and [CUDA driver](https://developer.nvidia.com/cuda-downloads).
 
-For an overview paper on PySDM v1 (and the preferred item to cite if using PySDM), see [Bartman et al. 2022](https://doi.org/10.21105/joss.03219) (J. Open Source Software).
+For an overview of PySDM features (and the preferred way to cite PySDM in papers), please refer to our JOSS paper:
+- [Bartman et al. 2022](https://doi.org/10.21105/joss.03219) (PySDM v1).
+- [de Jong et al. 2023](https://doi.org/10.21105/joss.04968) (PySDM v2).
 For a list of talks and other materials on PySDM, see the [project wiki](https://github.com/open-atmos/PySDM/wiki).
 
 A [pdoc-generated](https://pdoc3.github.io/pdoc) documentation of PySDM public API is maintained at: [https://open-atmos.github.io/PySDM](https://open-atmos.github.io/PySDM) 
 
+## Example Jupyter notebooks (reproducing results from literature):
```

Comment:
I suggest we move text about PySDM-examples into ./examples/README.md (similar to how this was before) and put a link to it here

Reply:
Good point (as it is also used by setup.py to prepare package description for pypi.org/p/PySDM-examples).
Addressed: https://github.com/open-atmos/PySDM/pull/1083/commits/f5ffa332207ec4fd601d4e8acc0a1352367d9d11